### ISSUE TITLE
Submenu in dropdown component isn't accessible from keyboard

### ISSUE THREAD

**Issue by vvtimofeev** [gravity-ui/uikit#1564]
We cannot open the second level of items in the dropdown component using keyboard input, it's only accessible with mouse. For example, you can see this [here](https://gravity-ui.com/components/uikit/dropdown-menu). Therefore, please add support of keyboard import there. 

<img width="1653" alt="Screenshot 2024-05-03 at 10 51 36" src="https://github.com/gravity-ui/uikit/assets/108340247/27ea24c7-fc93-48b1-9689-6a1d6d9be0ce">

**Comment by amje** [comment]
Will be fixed in #1430
